logging/efk-logging-curator.adoc

```diff
@@ -1,20 +1,28 @@
 :context: efk-logging-curator
 [id='efk-logging-curator']
-= Working with Elasticsearch Curator in {product-title}
+= Configuring Elasticsearch Curator in {product-title}
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 As an administrator, you can use the built-in Elasticsearch Curator tool to configure scheduled Elasticsearch maintenance operations
-to be performed automatically on a per-project basis.
+to be performed automatically on a per-project basis. Curator performs actions daily based on its configuration. Only one Curator Pod is
```

Comment:
Where can I find configuration for the cron schedule?

Reply:
We seem to link to Wikipedia (!):
link://en.wikipedia.org/wiki/Cron

Reply:
I should have thought about it more... I'm still mentally stuck with our ansible configuration, which we no longer use...
We used to have ansible variables that users could optionally use to control the cron schedule. Eg tell the installer to run curator every day at 2:30am.

With operators we instead tell users to modify the cronjob directly. The cluster logging operator does not watch for changes to that, which means that the operator doesn't revert these changes.

In other cases it might be quite frustrating, when the operator is in managed state and you make a change to a config option, which the operator watches for. In this case the operator would immediately revert such change and people might not notice at first..

Reply:
_In managed state, the Cluster Logging Operator reverts changes made to the `logging-curator` configuration map._